ksql-engine/src/test/java/io/confluent/ksql/embedded/KsqlContextTest.java

```diff
@@ -179,10 +181,10 @@ public void shouldTryExecuteStatementsReturnedByParserBeforeExecute() {
 
     // Then:
     final InOrder inOrder = inOrder(ksqlEngine, sandbox);
-    inOrder.verify(sandbox).execute(eq(STMT_0_WITH_SCHEMA));
-    inOrder.verify(sandbox).execute(eq(STMT_1_WITH_SCHEMA));
-    inOrder.verify(ksqlEngine).execute(eq(STMT_0_WITH_SCHEMA));
-    inOrder.verify(ksqlEngine).execute(eq(STMT_1_WITH_SCHEMA));
+    inOrder.verify(sandbox).execute(eq(sandbox.getServiceContext()), eq(STMT_0_WITH_SCHEMA));
```

Comment:
fyi, the `eq` calls are superfluous:

```suggestion
    inOrder.verify(sandbox).execute(sandbox.getServiceContext(), STMT_0_WITH_SCHEMA);
```

No need to change, just pointing it out.

Reply:
Changed anyway in the PR that removes the times(3)